lib/config-schema.coffee

```diff
@@ -38,6 +38,11 @@ module.exports =
         description: "Turn the particles on/off."
         type: "boolean"
         default: true
+      colours:
+        title: "Colours - Enabled"
+        description: "Turn colours on/off"
+        type: "boolean"
+        default: false
 
```

Comment:
this should be an enum type, given I want to add another option later for a fixed color.